Makefile

```diff
@@ -37,14 +37,15 @@ linux-main:
 		--sources ./Tests/ \
 		--templates ./.sourcery-templates/LinuxMain.stencil \
 		--output ./Tests/ \
-		--args testimports='$(imports)' \
-		&& mv ./Tests/LinuxMain.generated.swift ./Tests/LinuxMain.swift
+		--args testimports='$(imports)'
+	mv ./Tests/LinuxMain.generated.swift ./Tests/LinuxMain.swift
 
 route-partial-iso:
+	mkdir -p ./Sources/PointFree/Generated
 	sourcery \
 		--sources ./Sources/PointFree/ \
 		--templates ./.sourcery-templates/RoutePartialIsos.stencil \
-		--output ./Sources/PointFree/
+		--output ./Sources/PointFree/Generated/DerivedPartialIsos.swift
```

Comment:
moving code gen files to a `Generated` directory. was hoping I could use `.generated`, but SPM seems to ignore hidden directories.

Reply:
Could also do `__Generated__/`!

Reply:
ah duh! will do!